extensions/trello/src/utils/fetchTodos.tsx

```diff
@@ -0,0 +1,39 @@
+import { getPreferenceValues, showToast, ToastStyle } from '@raycast/api'
+import fetch from 'node-fetch'
+import { TrelloFetchResponse } from '../trelloResponse.model'
+const TRELLO_APP_KEY = "13f8c59607ba6d82531d3db5f46999c1";
```

Comment:
This is OK to be public

Reply:
Should use preferences

in package.json
```json
"preferences": [
    {
      "name": "token",
      "type": "password",
      "required": true,
      "title": "API Token",
      "description": "Your Trello API token taken from xxxx",
      "link": "https://github.com/xxx/xxxx",
      "placeholder": "Enter your Trello API token."
    }
]
...

then in utils.ts

```typescript
import { getPreferenceValues } from "@raycast/api";

interface Preferences {
  token: string;
  teamId: string;
  spaceId: string;
}

const preferences: Preferences = getPreferenceValues();

export default preferences;

```